tests/test_parsers.py

```diff
@@ -62,7 +62,11 @@ def test_parse(self):
         self.stream.seek(0)
         data_and_files = parser.parse(self.stream, None, self.parser_context)
         file_obj = data_and_files.files['file']
-        assert file_obj._size == 14
+        try:
+            size = file_obj.size
+        except AttributeError:  # before Django 2.1 _size was private
```

Comment:
I guess the relevant change is https://github.com/django/django/commit/a5406fe989d27598652dfd9f8dc1429b5d2db85c. I think `size` would work in older Django versions.